lib/HttpClient.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace Cronitor;
+
+class HttpClient
+{
+    public function __construct($baseUrl, $apiVersion = 3)
+    {
+        $this->baseUrl = $baseUrl;
+        $this->apiVersion = $apiVersion;
+    }
+
+    public function get($path, $params = array())
+    {
+        return $this->request($path, 'GET', $params);
+    }
+
+    public function delete($path, $params = array())
+    {
+        return $this->request($path, 'DELETE', $params);
+    }
+
+    public function put($path, $body = array(), $params = array())
+    {
+        return $this->request($path, 'PUT', $params, $body);
+    }
+
+    private function request($path, $httpMethod, $params = array(), $body = null)
+    {
+        $options = array(
```

Comment:
I opted for the built-in cURL interface so we didn't need to require a dependency of another http library, and all the version maintenance that comes along with it.

Reply:
Might be worth adding to required dev

```json
"ext-curl": "*",
```